src/panels/developer-tools/template/developer-tools-template.ts

```diff
@@ -0,0 +1,280 @@
+import "@material/mwc-button/mwc-button";
+import { UnsubscribeFunc } from "home-assistant-js-websocket";
+import {
+  css,
+  CSSResultArray,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+} from "lit-element";
+import { classMap } from "lit-html/directives/class-map";
+import { debounce } from "../../../common/util/debounce";
+import "../../../components/ha-circular-progress";
+import "../../../components/ha-code-editor";
+import { subscribeRenderTemplate } from "../../../data/ws-templates";
+import { haStyle } from "../../../resources/styles";
+import { HomeAssistant } from "../../../types";
+
+const DEMO_TEMPLATE = `{## Imitate available variables: ##}
+{% set my_test_json = {
+  "temperature": 25,
+  "unit": "°C"
+} %}
+
+The temperature is {{ my_test_json.temperature }} {{ my_test_json.unit }}.
+
+{% if is_state("sun.sun", "above_horizon") -%}
+  The sun rose {{ relative_time(states.sun.sun.last_changed) }} ago.
+{%- else -%}
+  The sun will rise at {{ as_timestamp(strptime(state_attr("sun.sun", "next_rising"), "")) | timestamp_local }}.
+{%- endif %}
+
+For loop example getting 3 sensor values:
+
+{% for states in states.sensor | slice(3) -%}
```

Comment:
This should not be the sensor domain.
On a new installation, we do not have _any_ sensor entities.
(we have that issue now as well, but since this is changing anyway, this should also change.)

Since you are slicing it anyway, it should be OK to select from all states?